tests/Microsoft.Identity.Web.Test/WebAppExtensionsTests.cs

```diff
@@ -411,6 +422,63 @@ public void AddMicrosoftIdentityWebApp_AddsInMemoryTokenCaches()
             Assert.Contains(services, s => s.ServiceType == typeof(IMsalTokenCacheProvider));
         }
 
+        [Fact]
+        // Regression for https://github.com/AzureAD/microsoft-identity-web/issues/1163
+        public void AddMicrosoftIdentityWebApp_CaseInsensitive_AppServices()
+        {
+            try
+            {
+                // Arrange
+                Environment.SetEnvironmentVariable(
+                    AppServicesAuthenticationInformation.AppServicesAuthEnabledEnvironmentVariable, "tRue");
+
+                Environment.SetEnvironmentVariable(
+                    AppServicesAuthenticationInformation.AppServicesAuthIdentityProviderEnvironmentVariable,
+                    "azureactivedirectory"); // used to be uppercase
+
+                var services = new ServiceCollection();
+
+                // Act
+                services.AddAuthentication(OpenIdConnectDefaults.AuthenticationScheme)
+                    .AddMicrosoftIdentityWebApp(_configureMsOptions);
+
+                // Assert
+                Assert.Contains(services, s => s.ServiceType == typeof(AppServicesAuthenticationHandler));
+            }
+            finally
+            {
+                ResetAppServiceEnv();
+            }
+        }
+
+        [Fact]
+        public void AddMicrosoftIdentityWebApp_AppServices()
+        {
+            try
```

Comment:
combine these two tests into one and use the `[Theory]
 [InlineData(string appServicesEnvEnabled, string idpEnvValue)]`